bug: Docker build fails due to missing exceljs and pdfkit dependencies

## 🐛 Bug Description

The `controls` service Docker build fails because `exceljs` and `pdfkit` are imported but not declared in `package.json`.

## 🛠 Steps to Reproduce

1. Clone the repository fresh
2. Run `./start.sh`
3. Build fails at `controls builder` step

## 📋 Expected Behavior

`./start.sh` completes successfully with all containers running.

## 💻 Environment Details

- **OS**: macOS / Linux
- **Docker**: Docker Desktop
- **Node.js**: 20.x

## 📝 Additional Context

**Error output:**
```
error TS2307: Cannot find module 'exceljs' or its corresponding type declarations.
error TS2307: Cannot find module 'pdfkit' or its corresponding type declarations.
```

**Introduced in:** PR #55 (commit `2ab32ab`, 2026-01-26)

--- a/services/controls/package.json
+++ b/services/controls/package.json
@@ -37,8 +37,10 @@
     "class-validator": "^0.14.0",
     "compression": "^1.7.4",
     "date-fns": "^4.1.0",
+    "exceljs": "^4.4.0",
     "helmet": "^7.1.0",
     "nodemailer": "^7.0.11",
+    "pdfkit": "^0.16.0",
     "prom-client": "^15.1.3",
     "reflect-metadata": "^0.1.14",
     "rxjs": "^7.8.1"
@@ -52,6 +54,7 @@
     "@types/jest": "^29.5.11",
     "@types/multer": "^2.0.0",
     "@types/node": "^20.10.0",
+    "@types/pdfkit": "^0.13.9",
     "@typescript-eslint/eslint-plugin": "^6.14.0",
     "@typescript-eslint/parser": "^8.50.0",
     "eslint": "^8.56.0",

